component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/mobileconnect/MobileConnectAuthenticator.java

```diff
@@ -729,7 +800,7 @@ protected String getCallbackUrl(Map<String, String> authenticatorProperties) {
         mobileConnectAcrValues.setName(MobileConnectAuthenticatorConstants.MOBILE_CONNECT_AUTHORIZATION_ACR_VALUES);
         mobileConnectAcrValues.setDisplayName("Mobile Connect ACR Values");
         mobileConnectAcrValues.setRequired(true);
-        mobileConnectAcrValues.setValue("1");
+        mobileConnectAcrValues.setValue("2");
         mobileConnectAcrValues.setDescription("Enter the Mobile Connect ACR Values required");
```

Comment:
Declare a constant and use here. In case you decide to change anything will be easier.